test/integration/user-management-api/buyers_application_keys_test.rb

```diff
@@ -83,7 +83,25 @@ def setup
                        method: '_destroy',
                        format: :xml })
 
-      assert_response :success
+      assert_response :success, "response #{@response.response_code} for key #{rm_key}"
+    end
+  end
+
+  # FYI this is the only way to delete keys ending in .xml or .json
```

Comment:
What do you mean "the only way"? They can also be deleted via UI.

Reply:
via API :smile:

Reply:
@mayorova is wording good now?